src/LoadingSkeleton/LoadingSkeleton.jsx

```diff
@@ -8,10 +8,11 @@ import colors from '../Styles/colors/palette';
 
 import './LoadingSkeleton.scss';
 
-const LoadingSkeleton = ({ className, ...props }) => (
+const LoadingSkeleton = ({ className, containerClassName, ...props }) => (
   <SkeletonTheme baseColor={colors.UX_GRAY_300}>
     <Skeleton
       className={classNames('LoadingSkeleton', className)}
+      containerClassName={classNames('loadingSkeletonContainer', containerClassName)}
```

Comment:
I sorta feel like we shouldn't expose containerClassName as a prop here but keep the fix.  In an ideal world, folks in in our private repo shouldn't have to worry about this inner container any longer:
```suggestion
const LoadingSkeleton = ({ className, ...props }) => (
  <SkeletonTheme baseColor={colors.UX_GRAY_300}>
    <Skeleton
      className={classNames('LoadingSkeleton', className)}
      containerClassName={'loadingSkeletonContainer'}
```

Curious what others think?

Reply:
I agree with Kyle that it makes more sense to not expose `containerClassName` as a prop. I'd rather only expose what we need from `react-loading-skeleton`. If we were to ever want to switch to another third-party package for loading skeletons, we don't want to be locked into supporting their prop interface or bumping a major version.

Reply:
Well, we already expose that prop with `{...props}` (line 11 and 34 this file) so if we do 

```
containerClassName={'loadingSkeletonContainer'}
```

then you can still override the prop and you may unintentionally not use this default class. Plus, because we say that `containerClassName` by default is `undefined` on line 64, it looks like the 

```
containerClassName={'loadingSkeletonContainer'}
```

will actually never be applied (wasn't able to get the default applied locally with suggested change).

But if you do this

 ```
containerClassName={classNames('loadingSkeletonContainer', containerClassName)}
```

then you can just apply your containerClassName (if you need/want to) while keeping the default class. 

For components that are separate packages that are very lightly wrapped by our DS, I'd vote to keep flexibility with `{...props}` (or naming those props, but still the same idea). Otherwise, we can't use what's available to us.

Lastly, we already use that prop in RS in e.g. `app/javascript/researcher/participant_profile/components/project_workspace_sessions/components/session_edit_form.jsx:43`

Reply:
If you flipped the order to be

```
    <Skeleton
      className={classNames('LoadingSkeleton', className)}
      {...props}
      containerClassName="loadingSkeletonContainer"
    />
```

then this would work, but then again you're losing ability to use this prop even though we have advertised that prop to be working since the inception of this component. I think that this

```
containerClassName={classNames('loadingSkeletonContainer', containerClassName)}
```

is the best solution

Reply:
All good points Dom

> Well, we already expose that prop with {...props} (line 11 and 34 this file)

You're right. Due to this we can't remove `containerClassName` as a prop without bumping a major version anyway. Like you said: they would be able to use it anyway, its in the propTypes, it is already used in RS.

> For components that are separate packages that are very lightly wrapped by our DS, I'd vote to keep flexibility with {...props} (or naming those props, but still the same idea).

I disagree with spreading props in a JS file since then we're accepting anything. That's a mistake we made that has locked us in to supporting all props of the libraries we wrap. Naming them or converting it to TS so they are restricted by the type system I agree with you.

> Otherwise, we can't use what's available to us.

That is intended. We should expose the props that we plan to support and encourage, not all the props that a third-party component has. Just because we're wrapping that component now, doesn't mean we'll use that third-party component forever. We should be free to replace third-party components as implementation details without affecting our own prop interface.

None of this is blocking for this PR. I'm talking about the "ideal world" Kyle's comment referred to and what I'd prefer in the future. As far as fixing the bug without introducing any breaking changes this PR nails it.

Reply:
> Otherwise, we can't use what's available to us.

> That is intended. We should expose the props that we plan to support and encourage, not all the props that a third-party component has. Just because we're wrapping that component now, doesn't mean we'll use that third-party component forever. We should be free to replace third-party components as implementation details without affecting our own prop interface.

If we wrap the 3rd party component like in this example and just pass all the props without really doing anything in between, then do we actually have our own prop interface for this component? Food for thought for the future ideal world. 

Seems like in those cases you want to be flexible (enabling using as many props as possible) since we're already making a decision to 'offload' this component completely to a 3rd party package. Plus, if you want to disable usage of certain number of props from a 3rd party component, then maybe it's a sign that we shouldn't pick that 3rd party dependency to begin with. 

I'm only mentioning this because this would make me worried that can we just slow each other down when we want/need to use a prop that's provided by a 3rd party.

Reply:
Let's just remove `containerClassName` from the props for now then since it's already exposed to rails-server via the spread props. That way at the least we aren't exposing multiple ways to control the same attribute.

Once that's done I'll merge this in.

Regarding the discussion regarding what of 3rd party code we should or should not expose as props, like Gabe I'm of the opinion that the less is best.  Lets move that convo to slack :)